src/native/libs/System.Globalization.Native/pal_locale.m

```diff
@@ -41,6 +42,505 @@
     return strdup(value);
 }
 
+/**
+ * Useful constant for the maximum size of the whole locale ID
+ * (including the terminating NULL and all keywords).
+ */
+#define FULLNAME_CAPACITY 157
+
+int static strnicmp(const char *str1, const char *str2, uint32_t n) {
```

Comment:
we don't need this function since Apple has one that does pretty much the same called `strncasecmp` https://developer.apple.com/library/archive/documentation/System/Conceptual/ManPages_iPhoneOS/man3/strcasecmp.3.html